gradlew.bat

```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
+@rem ##########################################################################
+@rem
+@rem  Gradle startup script for Windows
```

Comment:
do we need this? where does this come from?

Reply:
Gradle generates it – I'm not expecting people to be compiling this in Windows, so we can remove